base/global/thanos-query.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  labels:
+    kubernetes.io/ingress.class: traefik-private
+  name: global-thanos-query
+  namespace: sys-mon
```

Comment:
I think base should not contain any namespace definitions - happy to discuss.

Reply:
@george-angel you're absolutely correct, I'll move this into patches :+1: